src/System.Private.CoreLib/shared/System/Collections/Generic/Dictionary.cs

```diff
@@ -896,10 +905,10 @@ public bool TryGetValue(TKey key, out TValue value)
             int i = FindEntry(key);
```

Comment:
We should add TODO-NULLABLE-GENERIC to the signature, as the out TValue value could be null if TryGetValue returns false.